test/hotspot/jtreg/vmTestbase/nsk/share/Log.java

```diff
@@ -282,9 +232,6 @@ public synchronized void println(String message) {
      */
     @Deprecated
     public synchronized void comment(String message) {
-        if (!verbose()) {
-            doPrint(message);
-        }
```

Comment:
Is this method ever called? Is there a CR to remove it (and any references to it)?

Reply:
These are deprecated methods and are still getting called. The calls need to be replaced with display. I will create a separate CR for this and address because the impact radius of such a change is bigger than this CR.

Reply:
ok